scss/components/side-nav.scss

```diff
@@ -124,12 +124,31 @@ $block: #{$fd-namespace}-side-nav;
 
     &__icon {
         padding-right: $fd-side-nav-icon-right-padding;
+        margin-top: -1px;
+
+        &.sap-icon--s {
+            margin-top: 0;
+        }
+
+        &.sap-icon--m {
+            margin-top: 0;
+        }
+
+        &.sap-icon--l {
+            margin-top: -2px;
+        }
+
+        &.sap-icon--xl {
+            margin-top: -4px;
+        }
     }
 
     &--icons {
-        min-width: auto;
+        min-width: 75px;
         .#{$block}__icon {
             padding-right: 0;
+            position: relative;
+            left: calc(50% - 4px);
```

Comment:
When is this class used?

Reply:
this is for Side navigation collapsed with icons, if there is no that two lines code,
please see screenshot:
<img width="132" alt="Screen Shot 2019-06-27 at 4 50 48 PM" src="https://user-images.githubusercontent.com/41170611/60299882-c02e8c00-98fb-11e9-9028-e24e26d7e145.png">

Reply:
This entire `&--icons` block is sort of messed up.  It's basically like band-aids and top of band-aids, but never really addresses the primary issue.  I think that entire block can be replaced with this:
```scss
    &--icons {
        min-width: auto;

        .#{$block} {
            &__link,
            &__sublink {
                padding: fd-space(2.5) fd-space(5);
                justify-content: center;
            }

            &__icon {
                padding-right: 0;
            }
        }
    }
```

Rationale:
* Keep the `min-width` set to `auto` so it sizes to the width of its content.
* The padding on links and sublinks needs to be reset so it's the same on the left and right therefore content can be centered without `calc` statements.
* Change the flex box `justify-content` for links to force icons to be centered.
* Keep the removal of `padding-right` for icons.